cypress/e2e/generate_prompt.cy.js

```diff
@@ -74,8 +75,9 @@ describe('Prompt generation', () => {
         cy.get('[data-testid="fab-button"]').click();
         // Asserts that there are 2 delete buttons (1 for each IOBox)
         cy.get(deleteSelector).should('have.length', 2);
-        // Clicks the first delete button
-        cy.get(deleteSelector).first().click({ force: true });
+        // Hovers and clicks the first delete button
+        cy.get('[data-testid="hover-area"]').first().realHover();
+        cy.get(deleteSelector).first().click();
```

Comment:
This is going to cause a merge conflict with the changes to this same cypress test in IOBoxButtons-2 branch. Changing line 78 `cy.get(deleteSelector).first().click();` => `cy.get(deleteSelector).first().children().first().click();` should hopefully prevent it.